### ISSUE TITLE
SQLITE_ENABLE_DBSTAT_VTAB

### ISSUE THREAD

**Issue by orsinium** [elixir-sqlite/exqlite#231]
When I try to query the [dbstat](https://www.sqlite.org/dbstat.html) table I get the following exception:

```
     ** (Exqlite.Error) no such table: dbstat
     SELECT * FROM dbstat;
```

My noob understanding of the issue is that exlite ships with a sqlite version inside that is built without the  "[SQLITE_ENABLE_DBSTAT_VTAB](https://www.sqlite.org/compile.html#enable_dbstat_vtab)" [compile-time option](https://www.sqlite.org/compile.html). Is there a way to set it? I tried to `export EXQLITE_USE_SYSTEM=1` and `export SQLITE_ENABLE_DBSTAT_VTAB=1` and then `mix deps.compile --force`, nothing worked so far.

**Comment by orsinium** [comment]
I also tried:

```
export EXQLITE_SYSTEM_CFLAGS=-DSQLITE_ENABLE_DBSTAT_VTAB=1
mix deps.compile --force 
```

**Comment by warmwaffles** [comment]
I also noticed that we do not use `EXQLITE_SYSTEM_CFLAGS` unless we undefine `EXQLITE_USE_SYSTEM`. So that will need to be changed.

@orsinium let me get this in and I can cut a `v0.13.1` release

**Comment by warmwaffles** [comment]
@orsinium released under `v0.13.1`

**Comment by orsinium** [comment]
It works! Thank you :)
